keras_nlp/models/bert.py

```diff
@@ -0,0 +1,287 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Bert model and layer implementations."""
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers import PositionEmbedding
+from keras_nlp.layers import TransformerEncoder
+
+CLS_INDEX = 0
```

Comment:
I am not sure if we should hardcode `CLS_INDEX` in this file, there are two options:
1) hardcode `CLS_INDEX`, and clearly document that BERT model must take `CLS_INDEX=0`.
2) make it an argument of `Bert`, and default to 0.

I would prefer option 2) to avoid hidden information on the user end.

Reply:
Also think 2. is best (and the name should be fully spelled out)

Reply:
OK, will do #2. `[CLS]` is part of Bert mythology so it would probably be more confusing to change. Everyone calls it the C-L-S token.